PocketKit/Sources/Textile/Views/Tags/TagsList.swift

```diff
@@ -0,0 +1,26 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+public protocol TagsList {
+    func arrangeTags(with tags: [String]) -> [TagType]
+}
```

Comment:
Why is this a protocol?

Reply:
So that it can be used for both the view models for add tags (i.e. `PocketAddTagsViewModel`)  and edit/filter tags (i.e. `TagsFilterViewModel`)

Reply:
Protocols (imo) are for declaring abstract interfaces so outsiders don't need to know the concrete type they are interacting with.

If (as it seems) no one outside is trying to call `arrangeTags` then a protocol seems like an odd choice.

If what you wanted was for PocketAddTagsVM and TagsFilterVM to be able to arrangeTags, I would have gone with a simple helper function, potentially declared in the same file (but not the same Struct as TagType.